src/main/User/userRequests.js

```diff
@@ -17,11 +17,20 @@ export const createUser = (dispatch, history, email, password) => {
     })
     .catch(error => {
       console.log('FAIL: Failed to create new user with error: ', error.message)
-      dispatch({
-        type: TYPES.SET_ERROR_MESSAGE,
-        payload: {
-          error: 'Email already in use. Please try again',
-        },
-      })
+      if (error.response.status === 409) {
+        dispatch({
+          type: TYPES.SET_ERROR_MESSAGE,
+          payload: {
+            error: 'Email already in use. Please try again',
```

Comment:
let's use the error provided from the back end?

Reply:
where are the tests?

Reply:
fixed!